docs/meetings/client/2023_01_27.md

```diff
@@ -0,0 +1,80 @@
+# Client Meeting Minutes
+
+- **Start Time:** 4:00 PM
+- **End Time:** 4:30
+- **Location:** Zoom
+- **Date:** Jan 27th, 2023
+
+## Attendance
+
+- Client:
+  - Nazim
+- Team A:
+  - Veronica Jack
+  - Edouard Eltherington
+  - Logan Parker
+  - Matthew Kuelker
+
+### Attendance Comments
+
+- Other teams present (unclear if all team members were present)
+
+## Team A
+
+- Matt opened the boy demo for Nazim
+
+- Matt shared the Demo:
+  - There seemed to be un-synched lip positions
+  - We used challenge words and maybe some lag from zoom call
+
+- Logan ask Naz about requesting more poses:
+  - Nazim response: 'Its up to you and your creativity
+- Background should remain transparent
+
+- Matt and Logan brought up the language recognizers
+  - Challenge 1: English rec looks better, but takes longer to process
+
+Would you prefer effciency or quality ?
+
+- Naz says that quality is critical
+
+- Logan drops to clips in the discord chat
+  - One with english rec
+  - One with non-english rec
+
+- In terms of implementation Kuk uses a Docker server
+  - Logan mentions that we have a bottleneck in our local Dock implementaion

```

Comment:
This could honestly just be because of the way we're splitting it, they might have ways of handling this and they also said it's not necessary for us to dockerize it